heron/api/src/scala/Builder.scala

```diff
@@ -0,0 +1,46 @@
+//
```

Comment:
**General Comment:**
Currently, All files look living under `heron/api/src/scala`. 
- Related `*.scala` files should be under package: `heron/api/src/scala/com/twitter/heron/streamlet/scala`  (except `BUILD` file)
e.g: Currently, _Java Streamlet API_ lives under `heron/api/src/java/com/twitter/heron/streamlet`